components/dashboard/conf/Caddyfile

```diff
@@ -7,29 +7,38 @@
 	encode zstd gzip
 }
 
-(caching) {
-	header Cache-Control "public, max-age=604800, must-revalidate"
-}
-
-(default) {
-	header -Server
-}
-
 (discard_log) {
 	log {
 		output discard
 	}
 }
 
 :80 {
-	import caching
+	@static {
```

Comment:
Does it ensure that 404 won't be cached? Should we do something like here https://caddy.community/t/correct-way-to-set-expires-on-caddy-2/7914/13?